src/Grid.hpp

```diff
@@ -0,0 +1,104 @@
+//
+//  Grid.hpp
+//  Game of Life
+//
+//  Created by Alex Frasca on 01/05/2023.
+//
+#pragma once
+
+#include <SFML/Graphics.hpp>
+#include <vector>
+#include <cmath>
+#include <functional>
+
+namespace conway {
+
+//several values so we can draw nice colors as the cells die instead of just on/off
+enum class CELL_STATE {
```

Comment:
You sure you want all caps for enums and their values? Screaming case isn't very common in C++.

Reply:
I'm not sure why I got in the habit of doing it this way, but it's what I'm used to. what's the standard?

Reply:
Treat the enum like any other type, so same casing as a class/struct. Values can be all lowercase if you like, since they are all strongly scoped. However some values might clash with keywords like `or`, `float`, etc, so I use `eOr`, some folks use `kOr`, etc.